odd-platform-ui/src/components/DataEntityDetails/Overview/OverviewDataQualityReport/OverviewDataQualityReportStyles.ts

```diff
@@ -1,122 +1,57 @@
-    }) => {
-      // Stretching bars if total width less than 50%
-      const succRelation =
-        (datasetQualityTestReport?.successTotal || 0) /
-        (datasetQualityTestReport?.total || 1);
-      const otherStatusesAdjustment =
-        200 / (Math.round(1 - succRelation) + 1);
-      return {
-        height: '8px',
-        width: '100%',
-        [`&.${DataQualityTestRunStatusEnum.SUCCESS}`]: {
-          backgroundColor: theme.palette.runStatus.SUCCESS,
-          maxWidth: `${
-            (succRelation * 200) / (Math.round(succRelation) + 1)
-          }%`,
-        },
-        [`&.${DataQualityTestRunStatusEnum.FAILED}`]: {
-          backgroundColor: theme.palette.runStatus.FAILED,
-          maxWidth: `${
-            ((datasetQualityTestReport?.failedTotal || 0) /
-              (datasetQualityTestReport?.total || 1)) *
-            otherStatusesAdjustment
-          }%`,
-        },
-        [`&.${DataQualityTestRunStatusEnum.BROKEN}`]: {
-          backgroundColor: theme.palette.runStatus.BROKEN,
-          maxWidth: `${
-            ((datasetQualityTestReport?.brokenTotal || 0) /
-              (datasetQualityTestReport?.total || 1)) *
-            otherStatusesAdjustment
-          }%`,
-        },
-        [`&.${DataQualityTestRunStatusEnum.SKIPPED}`]: {
-          backgroundColor: theme.palette.runStatus.SKIPPED,
-          maxWidth: `${
-            ((datasetQualityTestReport?.skippedTotal || 0) /
-              (datasetQualityTestReport?.total || 1)) *
-            otherStatusesAdjustment
-          }%`,
-        },
-        [`&.${DataQualityTestRunStatusEnum.ABORTED}`]: {
-          backgroundColor: theme.palette.runStatus.ABORTED,
-          maxWidth: `${
-            ((datasetQualityTestReport?.abortedTotal || 0) /
-              (datasetQualityTestReport?.total || 1)) *
-            otherStatusesAdjustment
-          }%`,
-        },
-        [`&.${DataQualityTestRunStatusEnum.UNKNOWN}`]: {
-          backgroundColor: theme.palette.runStatus.UNKNOWN,
-          maxWidth: `${
-            ((datasetQualityTestReport?.unknownTotal || 0) /
-              (datasetQualityTestReport?.total || 1)) *
-            otherStatusesAdjustment
-          }%`,
-        },
-      };
-    },
-  });
+export const Container = styled('div')(() => ({
+  width: '100%',
+  display: 'flex',
+  justifyContent: 'stretch',
+  alignItems: 'stretch',
+  justifySelf: 'stretch',
+  flexGrow: 1,
+}));
 
-export type StylesType = WithStyles<typeof styles>;
+export const CountLabel = styled(Typography, {
+  shouldForwardProp: propsChecker,
+})<{
+  $dataQualityTestRunStatus: DataQualityTestRunStatusEnum;
+}>(({ theme, $dataQualityTestRunStatus }) => ({
+  color: theme.palette.runStatus[$dataQualityTestRunStatus],
+}));
+
+export const Bar = styled('div', { shouldForwardProp: propsChecker })<{
+  $datasetQualityTestReport?: DataSetTestReport;
+  $dataQualityTestRunStatus: DataQualityTestRunStatusEnum;
```

Comment:
You type similar props to different components, so to avoid duplication you can create interface e.g.
` interface CountLabelProps { $testRunStatus: DataQualityTestRunStatusEnum; }`
and then you can create another interface `BarProps` and extends with CountLabelProps. Then you can use both interfaces in type styled components.